### PR TITLE
doc: add v1.1.1 changeLog

### PR DESCRIPTION
**Pull request by daskyrk**
#### What type of this PR

Add one of the following kinds:
/kind document

#### What this PR does / why we need it:
add v1.1.1 changeLog
change title from second level to third level to make changeLog more clear.

### CODE REVIEW

File: CHANGELOG/CHANGELOG-1.x.md [hunk 1]
```diff
@@ -16,11 +17,21 @@
 </tr>
 </table>
 
+# v1.1.1
+
+`2021-8-5`
+
+### Bug Fixes
+
+* EDAS shares the same Agent with K8s. ([#1277](https://github.com/erda-project/erda/pull/1277))
```
Comment (hoozecn) [inline CHANGELOG/CHANGELOG-1.x.md:26]: EDAS and K8S use the same agent now.

File: CHANGELOG/CHANGELOG-1.x.md [hunk 2]
```diff
@@ -16,11 +17,21 @@
...
+* Releases functions of cloud management platform for free users. ([#810](https://github.com/erda-project/erda-ui/pull/810))
```
Comment (hoozecn) [inline CHANGELOG/CHANGELOG-1.x.md:27]: Enable cloud management platform for free users.

File: CHANGELOG/CHANGELOG-1.x.md [hunk 3]
```diff
@@ -16,11 +17,21 @@
...
+* Fixed the issue that error occured when editing custom addon of extended service. ([#813](https://github.com/erda-project/erda-ui/pull/813))
```
Comment (hoozecn) [inline CHANGELOG/CHANGELOG-1.x.md:28]: Fix errors caused by editing custom addon of extended service

**Review by hoozecn — CHANGES_REQUESTED** [review]

File: CHANGELOG/CHANGELOG-1.x.md (hunk 3 above)
Comment (daskyrk) [reply CHANGELOG/CHANGELOG-1.x.md:28]: Emmm, these are all translated by documentation engineer.

**Review by hoozecn — APPROVED** [review]

**Pull request merged**